cpu/lpc11u34/periph/pwm.c

```diff
@@ -50,7 +50,7 @@ uint32_t pwm_init(pwm_t pwm, pwm_mode_t mode, uint32_t freq, uint16_t res)
     }
 
     /* power on and configure the timer */
-    pwm_poweron(pwm);
+    LPC_SYSCON->SYSAHBCLKCTRL |= pwm_config[pwm].clk_bit;
```

Comment:
Maybe a `_poweron` static function could be used to avoid code line duplication.